bosch_locator_bridge_utils/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>bosch_locator_bridge_utils</name>
+  <version>0.0.0</version>
```

Comment:
```suggestion
  <version>1.0.0</version>
```

Reply:
Not sure about the version numbers yet. The version numbers for the `bosch_locator_bridge` package are:
- 1.0.x for noetic
- 2.0.x for foxy
- 2.1.x for main